.eslintrc

```diff
@@ -1,24 +1,35 @@
 {
+  "env": {
```

Comment:
since we have many projects in javascript/typescript maybe we should centralized some common configs such as eslint.